1.3/Defs/Interactions.xml

```diff
@@ -883,12 +910,17 @@
 				<!-- ===Bad opinion of recipient=== -->
 				<li>r_logentry(INITIATOR_opinion&lt;0)->[bad_opinion]</li>
 					<li>bad_opinion->...</li>
-					<li>bad_opinion(INITIATOR_opinion&lt;-20)->Hey, [recipient] — do me a favor, just leave me alone...</li>
+					<li>bad_opinion->[thank].</li>
+					<li>bad_opinion(INITIATOR_opinion&lt;-20)->You're not going to make me feel any better, [RECIPIENT_nameDef].</li>
+					<li>bad_opinion(INITIATOR_opinion&lt;-20,INITIATOR_TRAIT=abrasive)->Get right [swearword]out of here, [RECIPIENT_nameDef], before I climb out of this sickbed and make you leave my own [swearword]self!</li>
```

Comment:
Didn't fully understand this one.

Reply:
Does it make more sense if I fill in the blanks?
"Get right f***ing out of here, Joe, before I climb out of the sickbed and make you leave my own damned self!"
The abrasive initiator is telling the recipient to leave her alone, or she'll injure him.
I agree, though, this is a bit clunky. I'll remove the line for now.